src/imio/smartweb/common/tests/test_caching.py

```
@@ -0,0 +1,133 @@
+        with patch.dict(os.environ, {"CACHING_SERVERS": ""}):
+            ban_physicalpath(self._make_request(http_host="www.example.com"), ("", "Plone", "en"))
+        mock_request.assert_not_called()
+
+    @patch("imio.smartweb.common.caching.requests.request")
+    @patch("imio.smartweb.common.caching.api.portal.get")
+    def test_uses_x_forwarded_host_as_host_header(self, mock_portal_get, mock_request):
+        mock_portal_get.return_value = self._make_portal()
+        request = self._make_request(x_forwarded_host="www.example.com", http_host="internal.host")
+        with patch.dict(os.environ, {"CACHING_SERVERS": "192.168.1.10"}):
+            ban_physicalpath(request, ("", "Plone", "en"))
+        args, kwargs = mock_request.call_args
```

Ruff (0.15.10)

[warning] Unpacked variable `args` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)